config/_default/menus/menus.zh.toml

```diff
@@ -95,7 +95,13 @@
 #   weight = 40
 #   identifier = "storage-providers-seal-workers"
 #   url = "/storage-providers/seal-workers"
- 
+# 
+#[[storage-providers]]
+#  name = "Advanced configurations"
+#  weight = 50
+#  identifier = "storage-providers-advanced-configurations"
+#  url = "/storage-providers/advanced-configurations"
+#
```

Comment:
Nice! I always forget about the ZH menu. Good job!

Reply:
@johnnymatthews Can you approve the PR if everything looks okay?